catalogue/webapp/pages/works.tsx

```diff
@@ -355,16 +340,29 @@ const Works = ({
   );
 };
 
-export const getServerSideProps = async (
-  ctx: Context
-): Promise<{ props: Props }> => {
-  const globalContextData = getGlobalContextData(ctx);
-  const params = WorksRoute.fromQuery(ctx.query);
-  const { enableColorFiltering } = ctx.query.toggles;
-  const _queryType = cookies(ctx)._queryType;
+export const t: GetServerSideProps<{ things: string }> = async context => {
+  console.info(context);
+
+  const things = await Promise.resolve('things');
```

Comment:
need this anymore?

Reply:
removed the whole thing, was doing some typechecking debugging.